sonatype.sbt

```diff
@@ -0,0 +1,22 @@
+// Your profile name of the sonatype account. The default is the same with the organization value
+sonatypeProfileName := "com.rallyhealth"
+
+// To sync with Maven central, you need to supply the following information:
+publishMavenStyle := true
+
+// publish to Maven Central
+ThisBuild / sonatypeCredentialHost := "s01.oss.sonatype.org"
+ThisBuild / publishTo := Some {
+  if (isSnapshot.value)
+    Resolver.url("Sonatype", url("https://s01.oss.sonatype.org/content/repositories/snapshots"))
```

Comment:
Q: `s01` looks like it might be one of many, is that overly specific? i.e. is this pointing at just one node in the cluster when it should be pointing at some proxy for the entire cluster?

Reply:
This is the only cluster I am authorized to publish to. I think they separate the repositories by user. Each node in the cluster migrates the artifacts to all nodes within about 10 minutes. You can search for these artifacts on https://search.maven.org/ once they have synced with Maven Central.